deployment/lifelines

```diff
@@ -12,7 +12,7 @@ $wait Inherit LIFELINES_SHOPPER role in ANONYMOUSE role
 give --role LIFELINES_SHOPPER read --plugin app
 
 $wait Please import all the lifelines data
-import lifelines
+import -p deployment/data/lifelines.zip
```

Comment:
where does this zip com from ?

Reply:
from your laptop or the fileserver ;-D. But I need to describe this in the $wait I think. I will change the file.